build.gradle

```diff
@@ -10,6 +10,7 @@ buildscript {
     ext.detektVersion = '1.15.0'
     ext.gutenbergMobileVersion = 'v1.66.0'
     ext.storiesVersion = '1.2.0'
+    ext.aboutAutomatticVersion = '16-91ead3828a2c14e27da4955b80c79f856f794601'
```

Comment:
Should this point to the main branch before we merge this?

Reply:
Yes